src/main/java/com/example/report/service/ReportService.java

```
@@ -494,6 +684,133 @@ public List<ReportDTO> getAllReports() {
         return reportMapper.entityToDto(reportRepository.findAll());
     }
 
+    public Workbook extractPaySlip(Long employeeId, Long reportId){
+        List<ReportEntry> allByEmployeeIdAndReportId = reportEntryRepository.findAllByEmployeeIdAndReportId(employeeId, reportId);
+        Employee employee = allByEmployeeIdAndReportId.get(0).getEmployee();
+        Report report = allByEmployeeIdAndReportId.get(0).getReport();
```

Let's fetch data about Employee and Report from corresponding repository (we should throw exceptions if report or employee are not found)